src/style.css

```
@@ -1,3 +1,90 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+:root {
+  --black: black;
+  --blue: #00acdc;
+}
+
 body {
+  background-color: #f0f0f0;
+  width: 100%;
   margin: 0;
 }
+
+.container {
+  justify-content: space-between;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.nav__item {
+  list-style-type: none;
+  margin-right: 20px;
+  height: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.nav {
+  height: 100%;
+  align-items: center;
+  display: flex;
+}
+
+.container__links {
```

and reset default margin
```suggestion
.nav__list {
```